script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -1802,47 +1933,49 @@ Today isn't even Christmas, so why―――
 [sha:51a62ff7960034abbd3aa1d0ceff27970bd5683c]{
 -- Page 507, Offset 34742.
 -- 先輩が俺を突き飛ばす。
-Senpai pushes me away.
+Senpai pushes me out of the way.
 }
 [sha:ff03753fd2b7c6b7fa8e20d98edbc993645654c4]{
 -- Page 507, Offset 34743.
 -- その直後―――
-Right after that―――
+As soon as she does―――
 }
 [sha:aadd294bfadc5de64abccc464becb2ca433209e7]{
 -- Page 508, Offset 34744.
 -- 鉄柵のような細長い凶器が、昆虫採集の標本のように、シエル先輩の体を貫いていた。
-A long, narrow weapon that resembles an iron fence skewers Ciel-senpai like she's an entomology specimen.
+A long, narrow weapon that looks like the spire of a wrought iron fence skewers Ciel-senpai like she's a specimen in a butterfly collection.
```

Comment:
Oh, and we might as well take this here.

I think "spire" is a bad fit. Obviously "fence" sound dumb, but like, them being, well, for fencing people in is the entire aesthetic. She straight up does that with them later on when Shiki fights a bunch of zombies. 

I'd say 'railing' is liable to sound less stupid. Since these are essentially individual _parts_ of a railing, perhaps calling them "rails" is appropriate.

>...that looks like a section of railing...

Reply:
I don't really agree - spire or finial would be the correct term for the uprights, pretty sure. Or fence  post, but that doesn't have the pointy aspect. Railing calls to mind the horizontal bits instead, and when she _does_ use them to fence him in they're vertical.

Reply:
Nobody calls the vertical parts of a fence "spires". The end at the top are called spires, but not the actual thing

Reply:
I mean, it looks like one of those decorative spires though - googling around, it seems that the vertical members are also referred to as pickets even when metal. At any rate, railing doesn't really make me picture the right thing, so if we can find something else that'd be preferred.

Reply:
After some ELITE investigation, I've learned that the word that exclusively means "vertical part of a fence" is pailing.

Reply:
alright well there we go. We gotta burn the SAT word budget on something

Reply:
I have to say it sort of comes as a surprise, because "pail" just means bucket.

But apparently it comes from "pale", so there.